pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -178,6 +180,17 @@ def create(self, request, path):
         }
         return Response(data, status=http_status.HTTP_202_ACCEPTED)
 
+    def _dispatch_import_collection_task(self, repository, temp_file_path, **kwargs):
+        """
+        Dispatch a Import Collection creation task.
+        """
+        kwargs["temp_file_path"] = temp_file_path
```

Comment:
This method would create its own `kwargs = {}`.

Reply:
Disregard, this caller needs to build the `kwargs` for the cases when extra args need to be passed.